src/cmd_line.c

```diff
@@ -8,15 +8,273 @@ See the file LICENSE for details.
 #include "console.h"
 #include "string.h"
 #include "testing.h"
+#include "iso.h"
 
 #define KEYBOARD_BUFFER_SIZE 256
+#define ROOT "/"
 
-//Privates
+char cur_path[256]; // The current working directory for the command line environment
+
+void move_up_directory(char *abs_path);
```

Comment:
I asked @rboccabe the same thing. We decided ultimately these do directly relate to the command line, though I think we'd be open to discussing it tomorrow in the meeting.